Sources/Subs.php

```diff
@@ -3291,7 +3291,7 @@ function parsesmileys(&$message)
 function highlight_php_code($code)
 {
 	// Remove special characters.
-	$code = un_htmlspecialchars(strtr($code, array('<br />' => "\n", '<br>' => "\n", "\t" => 'SMF_TAB();', '&#91;' => '[')));
+	$code = un_htmlspecialchars(strtr($code, array('<br>' => "\n", '<br>' => "\n", "\t" => 'SMF_TAB();', '&#91;' => '[')));
```

Comment:
Is this one right? Seems like it was correct as it was both `<br / >` and `<br>`

Reply:
possibly is, I just done a search :) ill fix now